layouts/auth.vue

```diff
@@ -32,38 +32,46 @@
 </template>
 
 <script lang="ts">
-import { Component, mixins } from 'nuxt-property-decorator'
-import BaseMixin from '~/mixins/BaseMixin'
+import { defineComponent, useContext, useRoute } from '@nuxtjs/composition-api'
+import { computed } from 'vue'
+import { useBase } from '~/composables/useBase'
 
-@Component({
-    components: {
-        cookiePolicy: () => import('~/components/cookie.vue'),
+export default defineComponent({
+    setup() {
+        const ctx = useContext()
```

Comment:
```suggestion
        const { $config } = useContext()
```
No need for full context

Reply:
done